templates/.github/dependabot.yml

```
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      # Available options: daily, weekly or monthly
+      interval: "daily"
+      # Has to be formatted as hh:mm
+      time: "06:06"
+      # List of time zones: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+      timezone: "Europe/Paris"
+    # Allow updates for Open Terms Archive engine
+    allow:
+      - dependency-name: "@opentermsarchive/engine"
+    # Open Terms Archive recommends to always add the collection maintainer team as a reviewer so they are notified of the update
+    reviewers:
+      - "OpenTermsArchive/contrib"
```

If I understand correctly, this is the only part that should vary between collections. If that is the case, use a token and make sure it is [replaced by the init script](https://github.com/OpenTermsArchive/template-declarations/blob/main/init.sh#L42-L43) 🙂 
```suggestion
      - "OpenTermsArchive/${instanceName}"
```